src/public/map.js

```diff
@@ -83,6 +84,7 @@ map.on('draw:created', function (e) {
   }
 
   drawnShapes.addLayer(layer);
+  doPost("http://localhost:8040/search.sjs", "name", displayGeoJSON, drawnShapes);
```

Comment:
instead of "http://localhost:8040/search.sjs", make it "/search.sjs". The protocol, server, and port will be assumed to be the same as where the html file came from and that way it will work when we deploy it to a non-local server.